code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -232,3 +232,7 @@
 	var/fire_luminosity = 0 //Luminosity of the current fire while burning
 
 	var/butchery_progress = 0
+
+	//Trap vars
+	var/trapchoices = list()
+	var/gastier = 5
```

Comment:
Also would you mind adding docs to these defines?
Something like this but with detail on what its used for

```suggestion
	/// different trap choices
	var/trapchoices = list()
	/// the level of gas trap
	var/gastier = 5
```

Reply:
ah yes, Ill also do this. Sorry about that!